server/cluster/cluster.go

```diff
@@ -848,17 +852,29 @@ func (c *RaftCluster) HandleStoreHeartbeat(heartbeat *pdpb.StoreHeartbeatRequest
 		return errors.Errorf("store %v not found", storeID)
 	}
 
+	limit := store.GetStoreLimit()
+	version := c.opt.GetStoreLimitVersion()
+	var op core.StoreCreateOption
```

Comment:
opt or option is better. op usually stands for operator

Reply:
fixed